### ISSUE TITLE
Retain cycle

### ISSUE THREAD

**Issue by pilot34** [team-supercharge/boa#2]
Hi!

I have some misunderstanding. In you code we have retain-cycle ViewController <-> Presenter. How do you avoid leaks?

```objective-c
@interface ExamplePresenter : NSObject <ExampleModuleInterface>

@property (nonatomic, strong) ExampleInteractor *interactor;
@property (nonatomic, strong) ExampleWireframe *wireframe;

@property (nonatomic, strong) UIViewController<ExampleViewInterface> *userInterface;

@end



@interface ExampleViewController : UIViewController <ExampleViewInterface>

@property (nonatomic, strong) id<ExampleModuleInterface> eventHandler;

// *** add UI events here

@end

```

**Comment by pilot34** [comment]
PS: found similar issues for tutorials:
https://github.com/mutualmobile/VIPER-TODO/issues/2
https://github.com/objcio/issue-13-viper/issues/4

**Comment by exoszajzbuk** [comment]
@pilot34 Thanks for pointing it out! The issue is fixed with eda0f7b and the updated gem is published with version 0.3.2

How long do you use boa in your projects? We're glad to receive any feedbacks! :)
